devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy

```diff
@@ -124,7 +137,7 @@ def parse(String description) {
 	} else {
 		log.warn "Unexpected event: ${map}"
 	}
-	log.debug "Description ${description} parsed to ${result}"
+	//log.debug "Description ${description} parsed to ${result}"
```

Comment:
Remove altogether? Or leave it uncommented - I don't think it hurts anything and can be useful